packages/block-library/src/button/test/__snapshots__/index.js.snap

```diff
@@ -23,7 +23,11 @@ exports[`core/button block edit matches snapshot 1`] = `
               contenteditable="true"
               data-is-placeholder-visible="true"
               role="textbox"
-            />
+            >
+              <br
```

Comment:
Out of curiosity - is it something that TinyMCE puts before it gets initialized?